### PR TITLE
Alerting: docs for Unified Alerting opt-in, index, difference-old-new

### PR DESCRIPTION
**Pull request by davidmparrott**
**What this PR does / why we need it**:

- Reverts changes to alerts index page to reflect dashboard alerts, not unified alerts
- Paste content previously in alerts index that relates to unified alerts into unified-alerts/index.md
- Create opt-in.md to describe how to enable and disable unified alerts
- populate difference-old-new.md with basics of new features
- add empty files as place holders for docs linked elsewhere

**Special notes for your reviewer**:
Several new files are empty. I created them so that the docs site builds locally and they will need to get filled out

### CODE REVIEW

File: docs/sources/alerting/difference-old-new.md [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
 
->**Note:** It is recommended to backup Grafana's database before enabling this feature.
-
-When the feature flag is enabled, dashboard alerting is disabled and dashboard alerts are migrated into the system. Going to "Alert List" will take you to the new system.
-
-Once you disable the new alters, all migrated and newly created alerts in the new system are deleted, and dashboard alerting will be enabled again.
-
-During beta, the migration of existing dashboard rules may change.
\ No newline at end of file
+## View and search for alerts from Prometheus
```
Comment (domasx2) [inline docs/sources/alerting/difference-old-new.md:14]: also create and edit alerts from Cortex or Loki!

File: docs/sources/alerting/unified-alerting/_index.md [hunk 2]
```diff
@@ -1,11 +1,55 @@
...
+
+## Alert tasks
+
+You can perform the following tasks for alerts:
+
+- [Create an alert rule]({{< relref "./create-alert-rule.md" >}})
+- [View existing alert rules and their current state]({{< relref "./view-alerts.md" >}})
+- [Test alert rules and troubleshoot]({{< relref "./troubleshoot-alerts.md" >}})
+- [Add or edit an alert contact point]({{< relref "./contact-points.md" >}})
+
```
Comment (domasx2) [inline docs/sources/alerting/unified-alerting/_index.md:27]: notification policies deserve an entry as well

File: docs/sources/alerting/difference-old-new.md (hunk 1 above)
Comment (domasx2) [reply docs/sources/alerting/difference-old-new.md:14]: also maybe worth mentioning possibility to manage cortex alertmanager & grafana multidimensional alerts?

Comment (davidmparrott) [reply docs/sources/alerting/difference-old-new.md:20]: Good callouts, @domasx2 Added some stuff for that

File: docs/sources/alerting/unified-alerting/_index.md (hunk 2 above)
Comment (davidmparrott) [reply docs/sources/alerting/unified-alerting/_index.md:29]: Also added silences

**Comment by davidmparrott** [comment]
Moved away from Unified Alerting verbiage to Legacy and Grafana 8 verbiage

File: docs/sources/alerting/unified-alerting/_index.md (hunk 2 above)
Comment (achatterjee-grafana) [reply docs/sources/alerting/unified-alerting/_index.md:29]: The consensus we reached is that we will use "alerts" (referring to new alerts) and legacy alerts (referring to old Grafana alerts). This information should be at the top level.

**Review by achatterjee-grafana — COMMENTED** [review]
Added a comment about the need to move new alerts to the alerts anding page. Also, move current information to legacy. Otherwise, the information looks good!

**Review by achatterjee-grafana — APPROVED** [review]
LGTM! We will make further changes in subsequent PRs.

**Pull request merged**